tests/testthat/test-deprecated.R

```diff
@@ -73,3 +73,13 @@ test_that("split_rows_by_multivar works", {
     tbl <- build_table(lyt, DM2)
     expect_true(TRUE)
 })
+
+
+test_that("add_colcounts works", {
+    tbl <- basic_table() %>%
+        add_colcounts() %>%
+        analyze("AGE") %>%
+        build_table(DM)
+    
+    expect_equal(tbl[1, 1, drop = TRUE], mean(DM$AGE))
```

Comment:
Please change this test so that it checks the two ways generate identical tables

```
 tbl <- basic_table() %>%
        add_colcounts() %>%
        analyze("AGE") %>%
        build_table(DM)

tbl2 <- basic_table(show_colcounts = TRUE) %>%
        analyze("AGE") %>%
        build_table(DM)
expect_true(identical(tbl1, tbl2))
```

Note I used `expect_true(identical(.,.))`, NOT `expect_identical`, because `expect_identical` actually doesn't do identical(don't ask me, ask Hadley...)